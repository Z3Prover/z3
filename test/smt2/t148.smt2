
; Copyright (c) 2015 Microsoft Corporation


(declare-datatypes (T) ((lst nil (const (car T) (cdr mylist)))))
