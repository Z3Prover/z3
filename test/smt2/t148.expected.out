(error "line 5 column 19: invalid datatype declaration, unknown sort 'mylist'")
